sql/parser/eval_test.go

```diff
@@ -250,8 +250,8 @@ func TestEvalExpr(t *testing.T) {
 		{`'2010-09-28'::timestamp`, `2010-09-28 00:00:00+00:00`},
 		{`('2010-09-28 12:00:00.1'::timestamp)::date`, `2010-09-28`},
 		{`'2010-09-28 12:00:00.1'::timestamp`, `2010-09-28 12:00:00.1+00:00`},
-		{`'2010-09-28 12:00:00.1+02:00'::timestamp`, `2010-09-28 10:00:00.1+00:00`},
-		{`'2010-09-28 12:00:00.1-07:00'::timestamp`, `2010-09-28 19:00:00.1+00:00`},
+		{`'2010-09-28 12:00:00.1+02:00'::timestamp`, `2010-09-28 12:00:00.1+02:00`},
```

Comment:
I actually like the idea of converting the timestamp to UTC immediately after parsing, and using UTC timestamps everywhere on the server side. It's just going to be easier to reason about. The above comment about Truncate() is a good example of why. That would mean eliminating mentioning UTC everywhere on the server code except in ParseTimestamp/ParseDate.